backend/zkppSalt.js

```diff
@@ -60,10 +63,23 @@ export const initZkpp = async () => {
   recordSecret = Buffer.from(hashStringArray('private/recordSecret', IKM)).toString('base64')
   challengeSecret = Buffer.from(hashStringArray('private/challengeSecret', IKM)).toString('base64')
   registrationSecret = Buffer.from(hashStringArray('private/registrationSecret', IKM)).toString('base64')
+  hashUpdateSecret = Buffer.from(hashStringArray('private/hashUpdateSecret', IKM)).toString('base64')
```

Comment:
Used for encrypting the update token.

Reply:
Are these strings like `'private/hashUpdateSecret'` something that should be put in `zkppConstants.js` too?

Reply:
Probably not, since they're only used here for initialisation purposes.

Reply:
They also have no particular significance with regard to the protocol.